Comp2023/src/main/java/frc/robot/Constants.java

```diff
@@ -148,13 +144,28 @@
       ELBOW_UP            // Move elbow up
     }
 
+    public enum ElbowAngle
+    {
+      ELBOW_NOCHANGE,     // No change in elbow Angle--maintain current position
+      ELBOW_STOW, // move arm to stow Angle
+      ELBOW_LOW, // move arm to low-scoring Angle
+      ELBOW_MID, // move arm to shelf Angle; slightly higher than mid-scoring Angle so this is used for both
+      ELBOW_HIGH, // move arm to high-scoring Angle
+    }
+
     public enum WristMode
     {
       WRIST_INIT,         // Initialize wrist
       WRIST_DOWN,         // Move wrist down
       WRIST_STOPPED,      // Stop and hold position
```

Comment:
Maybe rename this as STOP or HOLD to be consistent with Gripper and as it will be stopping?

Reply:
We used NOCHANGE or HOLD in the past for these states. Since the elbow and wrist run all the time, they will never really be "stopped". The motors will be powered even to hold these joints in the correct position.